styles/abstracts/_theme.scss

```diff
@@ -5,35 +5,43 @@ $themes: (
     'text-color': #000000,
     'border-color': #000000,
     'text-color-inverse': #ffffff,
+    'background-color': #ffffff,
+    'background-color-inverse': #191718,
     'k-accent': #ff7ac3,
     'k-accentlight': #ffe5f3,
-    'k-accentlight2': #fff0f8,
+    'k-accentHover': #ffe5f3,
+    'k-accentlight2': #ffe5f3,
     'k-green': #04af00,
     'k-red': #ff5757,
     'k-grey': #999999,
     'k-hovergrey': #6b6b6b,
     'k-blue': #6188e7,
```

Comment:
I'm thinking removing `k-green` `k-blue` etc from here, because they have the same color in both dark mode and light mode. We could use varable `$k-blue` instead. WDYT @roiLeo @prachi00

Reply:
yeah I thought that too but for different themes, like for different chains, what if we need a different color? then we'd have to do this again all over